storage/time-series/influxdb/retriever/app/influx/DataQuery.js

```diff
@@ -0,0 +1,272 @@
+/* eslint-disable no-underscore-dangle */
+const createError = require('http-errors');
+const {
+  InfluxDB,
+  flux,
+  fluxExpression,
+  fluxInteger,
+  fluxDateTime,
+  fluxString,
+} = require('@influxdata/influxdb-client');
+const util = require('util');
+const { Logger } = require('@dojot/microservice-sdk');
+
+const logger = new Logger('influxdb-retriever:influx/DataQuery');
+
+/**
+ * This class handle with query data in a specific bucket.
+ * @class
+ */
+class DataQuery {
+  /**
+   *
+   * @param {String} url   Url to access influxdb
+   * @param {String} token  A token with write permission in all orgs
+   * @param {String} defaultBucket  Bucket Name for all data write
+   *
+   */
+  constructor(url, token, defaultBucket) {
+    logger.debug('constructor:');
+    logger.debug(`constructor: url=${url}`);
+    logger.debug(`constructor: token=${token}`);
+    logger.debug(`constructor: defaultBucket=${defaultBucket}`);
+
+    this.influxDB = new InfluxDB({ url, token });
+    this.defaultBucket = defaultBucket;
+    // prefix adds to all fields to be written
+    this.prefixFields = 'dojot.';
+    this.prefixFieldsSize = (this.prefixFields).length;
+  }
+
+  /**
+ * Fetch data for a given field considering the time
+ * slot and paging filter for a default bucket and an given org.
+ *
+ * @param {String} org Organization Name
+ * @param {String} measurement Measurement Name
+ * @param {object} filters Filters for query
+ * @param {string} filters.dateFrom=1970-01-01T00:00:00.000Z
+ * @param {string} filter.dateTo=(current time)
+ * @param {object} page Paginate information for query
+ * @param {number} page.limit=256
+ * @param {number} page.page=1
+ * @param {{String='desc','asc'}} order=desc Defines whether the order by **time** should be
```

Comment:
the number of annotations in the documentation (9) does not match the number of parameters accepted by the function (5). Is this valid?

Reply:
Yes, this is a way that I could see the pattern of describing what is expected within the objects